src/pages/TeamSettings/Account/License.vue

```diff
@@ -21,10 +21,15 @@ export default {
     }
   },
   computed: {
-    ...mapGetters('license', ['license', 'planType', 'allowedUsers']),
+    ...mapGetters('license', [
+      'license',
+      'planType',
+      'allowedUsers',
+      'hasPermission'
+    ]),
     ...mapGetters('tenant', ['tenant']),
     isTenantAdmin() {
-      return this.tenant.role === 'TENANT_ADMIN'
+      return this.hasPermission('create', 'role')
```

Comment:
I don't think this is used in this component but probably should be changed to `update:license` and used to disable the cancel plan button